### DISCUSSION TITLE
Monthly dev call

### DISCUSSION THREAD

**Discussion by davehakkens** [ONEARMY/community-platform#1854] [General]
This topic is to arrange our monthly dev calls. We will share the upcoming agenda and tag people that have been actively participating as a reminder to join. 

It is open for everyone to join and each month we have a different topic. 
From frontend, backend, UI, UX design etc.

**Comment by davehakkens** [comment]
# 1. Design and Storybook
**Date:** 5th September 18:00 GMT
**Location:** [Discord](https://discord.gg/GypycJ79?event=1006608338513051769)
**Duration:** 1 Hour + Questions
 
Join our first public call for the Community Platform! It is for everyone that wants to help out with the development of our software. This month we will go over the Design and Storybook . @franknoirot  will explain you more on how to [design](https://onearmy.github.io/community-platform/Design/our-design-system) new elements using Figma and  @thisislawatts  will give a demo on how we work with components and [Storybook](https://onearmy.github.io/community-platform/storybook-static/index.html?path=/story/base-components-button--basic).

**[Dail in on Discord]( https://discord.gg/GypycJ79?event=1006608338513051769)** at 5th September 6pm to see the demo’s and ask questions along the way.  We’re planning for one meetup each month with a different theme.  So feel free to add suggestions.

Tagging you all to make sure you are aware of it :)
@chrismclarke @patrycjapraczyk @danitrod @jojudre @amuroBosetti @mef  @ryanwbass  @eliasvelardezft @tedspare @AlfonsoGhislieri @Xyli0 @laianbraum @osouthwell-scottlogic @asheerrizvi @LucasGabrielBecker @paulpichot @tiefpunkt @tudi2d 




<img width="532" alt="afbeelding" src="https://user-images.githubusercontent.com/13672737/184095318-5c4b8826-8ef7-439f-83fc-4b6f1401c8f9.png">

**Reply by davehakkens** [reply]
Watch back the meeting
https://youtu.be/IZtpvwuXlNQ

**Comment by davehakkens** [comment]
# 2. How to get started contributing code

**Date:** 3th October 18:00 GMT
**Location:** [Discord](https://discord.gg/GypycJ79?event=1006608338513051769)
**Duration**: 1 Hour + Questions

This monthly meetup @thisislawatts will show how to get started contributing to the Community Platform codebase. How to fork, test locally use the emulator, Docker and were to look out for when making a PR. He will have a few tips & tricks and ready to take on questions or help troubleshoot. 

[Dail in on Discord](https://discord.gg/M9eTUyCE?event=1022584600763052132) at 3th September 6pm to see the demo  and ask questions along the way.  We’re planning for one meetup each month with a different theme.  So feel free to add suggestions.

<img width="397" alt="afbeelding" src="https://user-images.githubusercontent.com/13672737/191830794-014954d9-9352-4788-922f-d763ca17290c.png">

@ONEARMY/contributors

**Reply by Sc4ramouche** [reply]
Hey @davehakkens! The discord invite link seems to be expired 👀

**Reply by davehakkens** [reply]
yeh we just finished the meeting :|

**Reply by Sc4ramouche** [reply]
ah, sorry. I was thinking I messed up my discord account so it took me a while to troubleshoot. Till next time then 👋

**Reply by davehakkens** [reply]
Watch back the meeting
 https://youtu.be/LUuNY-IMGHw

**Comment by davehakkens** [comment]
# 3. Developers Meetup Community Platform

Date: Monday 7th November 18:00 GMT
Location: [Discord](https://discord.gg/hv85YEBy?event=1036747806301491280)
Duration: 1 Hour + Questions

This month we have a casual meetup to meet the developers, maintainers and contributors behind the project. If you pushed code or are interested to help out, join in and have a chat. Also a good time to ask any questions and see who is behind those avatars:)

Might be nice to join @asheerrizvi @Sc4ramouche  @Kiebert  @Ulian  

<img width="532" alt="afbeelding" src="https://user-images.githubusercontent.com/13672737/199111343-5d47679f-8ff0-4237-a188-b4b275c42ea4.png">
